bin/gem2rpm

```diff
@@ -81,6 +81,8 @@ if options[:srpm]
     "--define '_srcrpmdir #{out_dir}' #{specfile}"
   unless system(command)
     Gem2Rpm.show_message("Command failed: #{command}")
+    content = Gem2Rpm.file_content(specfile)
```

Comment:
Sorry, but I don't like this behavior. There are plenty of other ways how to get the .spec file content.

Moreover, this is new feature and as such, it should not be part of this change.